database/query.go

```diff
@@ -63,8 +63,8 @@ var (
 	QuerySolvedChls = "SELECT solved_challenges FROM team WHERE tag=$1"
 	QueryEventTable = "SELECT * FROM event"
 
-	// finished_at '0001-01-01 00:00:00 means event does not finished yet '
-	QueryEventId    = "SELECT id FROM event WHERE tag=$1 and finished_at = date('0001-01-01 00:00:00'); "
+	// query only running, suspended and booked events
+	QueryEventId    = "SELECT id FROM event WHERE tag=$1 and finished_at = date('0001-01-01 00:00:00') and (status = 0 or status = 1 or status = 2)"
```

Comment:
For the sake of readability i would probably use `and status != 3` instead. Would yield the same result.